spacewalk/admin/uyuni-update-config

```diff
@@ -186,14 +188,32 @@ def change_billing_data_service():
         sys.stdout.write("billing-data-service sysconfig: changed LISTEN address\n")
 
 
+def hostname_check():
```

Comment:
This function should not be named 'check' when it change something.

Reply:
Fixed